ccdproc/tests/run_for_memory_profile.py

```diff
@@ -1,3 +1,6 @@
+import pytest
+pytest.importorskip('memory_profiler')
```

Comment:
I know it is listed as a test dependency but it is strictly not necessary unless someone runs into memory issues.

Reply:
Are you suggesting dropping from the test dependencies or dropping the line here?

Reply:
I mean you can still have it as test dependency, but I think it is nicer to let the rest of the suite run when it is not installed. But if you really don't like this skip, I can revert.

Reply:
🤦 please leave it in -- definitely a good idea.